### ISSUE TITLE
step by step train strategy

### ISSUE THREAD

**Issue by kepengxu** [JianpingLin/M-LVC_CVPR2020#9]
hello 
I implement train code for step-by-step train the model,
by when I train step 2 (train mvd-autoencoder and res-autoencoder) I fine the l apply to train_bpp will lead to PSNR of reconstruction flow warp image will be too low
I observe loss and find mse ≈ e-4 and bpp between 0 ~ 0.5, why use too large l to multiply bpp. 
 In total loss bpploss : mseloss ≈ 0.2: 0.0005.
thank you very much if you can introduce more detail of the training process.

**Comment by lwplw** [comment]
I'm glad to see that someone has reproduced the model training step. Do you understand the questions I asked in the issue, about #7 and #8, because I can't get the author's reply, I hope to get your help.
thank you vary much

**Comment by lwplw** [comment]
Can you share your training code when appropriate?

**Comment by kepengxu** [comment]
> Can you share your training code when appropriate?

I think you can check the shape of the tensor because this isn't a difficult problem to solve

**Comment by lwplw** [comment]
thanks!

I have located the #7 error in the original code.
Regarding #8, it seems that the `def decompress()` function in `entropy_models_gauss.py` needs to be rewritten. Of course, the problem should still be about the shape of the tensor.

**Comment by weihua04** [comment]
> hello
> I implement train code for step-by-step train the model,
> by when I train step 2 (train mvd-autoencoder and res-autoencoder) I fine the l apply to train_bpp will lead to PSNR of reconstruction flow warp image will be too low
> I observe loss and find mse ≈ e-4 and bpp between 0 ~ 0.5, why use too large l to multiply bpp.
> In total loss bpploss : mseloss ≈ 0.2: 0.0005.
> thank you very much if you can introduce more detail of the training process.

Hello, Do u plan to open source the training code?
